.devcontainer/devcontainer.json

```
@@ -4,7 +4,8 @@
   "name": "Pycytominer Dev",
   "image": "mcr.microsoft.com/devcontainers/python:0-3.11",
   "features": {
-    "ghcr.io/devcontainers-contrib/features/poetry:2": {}
+    "ghcr.io/devcontainers-contrib/features/poetry:2": {},
+    "ghcr.io/devcontainers/features/docker-in-docker": {}
```

Super minor: similar to the `poetry` feature, would it make sense to pin to the latest major version of this feature to help keep it explicit?
```suggestion
    "ghcr.io/devcontainers/features/docker-in-docker:2": {}
```